src/pages/Swap/index.tsx

```diff
@@ -625,28 +625,6 @@ export default function Swap() {
                     {/*        </DarkCard>*/}
                     {/*    </div>*/}
                     {/*)}*/}
-                    <a
```

Comment:
I think we should keep this concept and use it for advertising franchises/features